evidence/doc.go

```diff
@@ -0,0 +1,37 @@
+/*
+Package evidence handles all evidence storage and gossiping from detection to block proposal.
+For the different types of evidence refer to the `evidence.go` file in the types package
+or https://github.com/tendermint/spec/blob/master/spec/consensus/light-client/accountability.md.
+
+## Gossiping
+
+The core functionality begins with the evidence reactor (see reactor.
+go) which operates both the sending and receiving of evidence.
+
+The `Receive` function takes a list of evidence and does the following:
+ 1. breaks it down into individual evidence if it is `Composite Evidence` (see types/evidence.
+go#ConflictingHeadersEvidence)
```

Comment:
Two more formatting nitpicks: these points should be sentence-cased, and it's a bit funny to put the line break in the middle of the filename

Reply:
yeah lol, I did it previously because the yellow squiggly line from the linter was annoying me. But fixed it now.